client/src/pages/dashboard/stores/DashboardState.ts

```diff
@@ -0,0 +1,22 @@
+import { writable } from '@square/svelte-store';
+import { Office } from '~model/office';
+
+export interface dashboardStateModel {
```

Comment:
Prefer `PascalCase` for interfaces and type aliases. Note that the renamed `interface` should be propagated everywhere else.
```suggestion
export interface DashboardStateModel {
```